### DISCUSSION TITLE
Private video on Mobile App (Ionic Framework)

### DISCUSSION THREAD

**Discussion by josephlodero** [vimeo/player.js#694] [General]
does anybody here tried using this package with private video in ionic/mobile?

**Comment by ozimovski** [comment]
I've tried but no success so far. 

Really dirty fix for android was to set up a localhost domain in embedded specified domains settings.
Of course that doesn't work for iOS tho (iOS is running on capacitor://localhost).

**Reply by LorenzHope** [reply]
We have same issue, 
Added domain capacitor and ionic -> not working
Added full url with ?h=<...> -> not working
